### PR TITLE
Add a README to the React package

### LINKED ISSUE #3465: Add a README to the React package
The Spark package on npm's site should have a README. Can copy the existing README in the root and make edits to customize it for React. Make PR against staging

### PR DESCRIPTION
**Pull request by Ahmeed2m**
## What does this PR do?
This PR adds a README to the react package.
It's updated to fit the react version copy of the root README, you can find the diff from the root readme [here](https://gist.github.com/Ahmeed2m/db22474bce104b1d1cfe7617053f5f4a/revisions?diff=split&short_path=b335630)
I have used the [docs](https://sparkdesignsystem.com/installing-spark/react) installation guide to make the needed changes.

### Associated Issue
Fixes #3465

### Documentation
 - [x] Update Spark Docs

### CODE REVIEW

File: react/README.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+  When you build and run your website, all the Spark CSS will now be bundled into your output. You can verify this by inspecting your site’s DOM and looking for the Spark CSS. There should be a `<style>` tag in the `<head>` of your page that includes Spark’s styles.
+  The Spark classes
+  are namespaced so they don't affect any existing CSS.
+
+* Now you have Spark react installed and you can import tree-shaked components like Spark Button:
+
+  `import { SprkButton } from '@sparkdesignsystem/spark-react';`
+
+  You can find code for all Spark components at react.sparkdesignsystem.com.
```
Comment (guern1kn) [inline react/README.md:38]: Let's make this react.sparkdesignsystem.com a clickable link.

**Review by guern1kn — CHANGES_REQUESTED** [review]
Thanks for the PR, @Ahmeed2m ! One comment on there.

File: react/README.md (hunk 1 above)
Comment (Ahmeed2m) [reply react/README.md]: Should be fixed now

**Review by guern1kn — APPROVED** [review]
Thanks for the contribution, @Ahmeed2m !

**Pull request merged**